Java/avg/array.java

```diff
@@ -12,4 +12,15 @@ public static void main(String[] args) {
             System.out.println(arr[i]);
         }
     }
+
+    public static void Even(String[] args) {
+        Scanner sc = new Scanner(System.in);
+        int[] arr = new int[]{1, 2, 3, 4, 5};
+        System.out.println("Even numbers:");
+
+        for (int i = 0; i < arr.length; i = i * 2)

```

Comment:
**Optimization Issue**: The loop increment i = i * 2 in the 'Even' method results in an infinite loop when i is 0, leading to a severe performance issue. This can cause the application to hang or crash, especially when handling large arrays. <br> **Fix**: Change the loop increment to i += 2, which ensures the loop iterates through even-indexed elements without causing an infinite loop. <br> **Code Suggestion**: 
 ```
 for (int i = 0; i < arr.length; i += 2)
         {
             System.out.println(arr[i]);
         }
 ```